docs/tutorial.rst

```diff
@@ -289,116 +375,78 @@ This is useful for integrating with tools such as
 but note that what we call genotype matrix corresponds to a
 scikit-allel haplotype array.
 
-******************
-Historical samples
-******************
+.. _sec_tree_sequences:
 
-Simulating coalescent histories in which some of the samples are not
-from the present time is straightforward in ``msprime``.
-By using the ``samples`` argument to :meth:`msprime.simulate`
-we can specify the location and time at which all samples are made.
+**************************
+A bit about tree sequences
```

Comment:
This bit feels like it's more about "working with trees" than tree sequences?

Reply:
Yeah, I thought this too. I think it could be useful to have a *very* short and contained blurb about tree sequences here somewhere (way shorter than what is here), and then link to the tskit docs for users to learn more.

Reply:
maybe after this section, titled "The tree sequence's tables", saying basically that everything is stored in tables; go see tskit?